Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java

```diff
@@ -134,46 +199,79 @@ public void initDefaultCommand() {
 		setDefaultCommand(new TeleopDrive());
 	}
 
+	/**
+	 * Disables the turnPID PIDController used for turning
+	 */
 	public void disableTurnPid() {
 		turnController.disable();
 	}
 
+	/**
+	 * Enables the turnPID PIDController used for turning
+	 */
 	public void enableTurnPid() {
 		turnController.enable();
 	}
 
+	/**
+	 * Sets the setPoint of the turnPID PIDController
+	 * 
+	 * @param set
+	 *            The value to set the setPoint at
+	 */
 	public void setSetTurn(double set) {
 		turnController.setSetpoint(set);
 	}
 
-	// public void disableMoveLeftPid() {
-	// moveLeftController.disable();
-	// }
-	// public void enableMoveLeftPid() {
-	// moveLeftController.enable();
-	// }
-	// public void setSetMoveLeft(double set) {
-	// moveLeftController.setSetpoint(set);
-	// }
-	//
-	// public void disableMoveRightPid() {
-	// moveRightController.disable();
-	// }
-	// public void enableMoveRightPid() {
-	// moveRightController.enable();
-	// }
-	// public void setSetMoveRight(double set) {
-	// moveRightController.setSetpoint(set);
-	// }
+	/**
+	 * Enable the movePID PIDController used for moving
+	 */
+	public void enableMovePid() {
+		moveController.enable();
+	}
 
-	public void setDistanncePerPuls(boolean left, double dist) {
-		if (left) {
-			leftEnc.setDistancePerPulse(dist);
-		} else {
-			rightEnc.setDistancePerPulse(dist);
-		}
+	/**
+	 * Disables the movePID PIDController used for moving
+	 */
+	public void disableMovePid() {
+		moveController.disable();
+	}
+
+	/**
+	 * Sets the setPoint of the movePID PIDController
+	 * 
+	 * @param set
+	 *            The value to set the setPoint at
+	 */
```

Comment:
can the "move" PID in general be changed to "drive"? it's just a little clearer bc "move" can refer to turning to technically. I mean "drive" can as well but not as much